testapps/integration/web/index.php

```diff
@@ -29,12 +29,12 @@
 });
 
 $app->post('/logging_standard', function (Request $request) {
-    $logName = $request->get('log_name');
-    $token = $request->get('token');
+    $logName = $request->request->get('log_name');
```

Comment:
Ahh, sorry this is confusing. The `logging_standard` test doesn't make use of this field at all, even though it is actually provided by the client. I just combined the payloads of the two tests to simplify client code a bit. The standard logging tests should just be using the language's standard logging module to write the log entry, without specifying a name. Not all languages have this feature (not sure if PHP does; in python this is accomplished by simply doing `logging.info(token)`, for level `INFO`); if PHP doesn't have this, feel free to opt out of this test.